src/backend/expungeservice/database/db_util.py

```diff
@@ -12,3 +16,24 @@ def wrapper(database, *args, **kwargs):
             raise ex
 
     return wrapper
+
+
+def save_search_event(request_data, record):
+    user_id = current_user.user_id
+    search_param_string = user_id + json.dumps(request_data["aliases"], sort_keys=True)
+    hashed_search_params = hashlib.sha256(search_param_string.encode()).hexdigest()
+    _db_insert_search_event(g.database, user_id, hashed_search_params)
+
+
+@rollback_errors
+def _db_insert_search_event(database, user_id, hashed_search_params):
+    database.cursor.execute(
+        """
+        INSERT INTO  SEARCH_RESULTS(search_result_id, user_id, hashed_search_params)
```

Comment:
Do you not need to run migrations to delete the columns first? The old db will expect 5 entries but only receive 3. How will it know that the final 2 should be just null?

Reply:
Nah it just works, and I tested on both versions of the table. Those two columns are nullable, so because we specify to insert to only the 3 columns, it only expects those values.

Running migrations in tandem would be good practice, but not strictly necessary.